man/wacom.man

```diff
@@ -235,7 +235,10 @@ sets the list of serial numbered devices that need to be hotplugged for a physic
 device. The 'type' option may be any of "pen", "airbrush", "artpen", or "cursor".
 This option is only available on wacom V devices (Intuos series and Cintiq 21U).
 To see which serial number belongs to a device, you need to run the utility program,
-xsetwacom, that comes with this driver.
+xsetwacom, that comes with this driver. In layman's terms, this option will add X
+(virtual) slave pointers for this specific tool's tip (and eraser, if applicable).
```

Comment:
I'd just say "will add an X device for this specific tool", the exact approach how X handles the devices doesn't matter for this purpose.

Plus, we should avoid using the term `slave pointer` as much as possible, even though we'll likely never be able to get rid of it here.

Reply:
Fixed